src/study.js

```diff
@@ -265,27 +272,11 @@ export default class Study extends Component {
   }
 }
 
-class StudyLeadImage extends Component {
-  render() {
-    const { image } = this.props;
-    return (
-      <div
-        className="study__introduction__image"
-        style={{ backgroundImage: `url(${image})` }}
-      />
-    );
-  }
-}
-
 Study.propTypes = {
-  study: PropTypes.object.isRequired,
+  study: PropTypes.shape.isRequired,
```

Comment:
This is a nice feature that I didn't know about. Should the particular shape be referenced?

Reply:
Yes you're right, this hasn't been completed properly. I don't think I had worked out what the shape should be yet.

Reply:
I've added the shape now from observing a console log.